telemetry/data_feature_usage.go

```diff
@@ -28,12 +28,15 @@ import (
 	"github.com/tikv/client-go/v2/metrics"
 )
 
+var Usage = ClusterIndexUsage{}
```

Comment:
```suggestion
// emptyClusterIndexUage is (for a deprecated)...
var emptyClusterIndexUsage = ClusterIndexUsage{}
```
1. It's fine to make it private since you can still get it from `telemetry_test` package.
2. Please add a comment for it.

Reply:
very good advice.